content/blog/2023/knight_odds.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Play with knight odds against Lc0 on lichess"
+author: "Naphthalin"
+published: 2023-11-07
+---
+
+In the recent [blog post]({{< relref "wdl_model.md" >}}) presenting the new WDL contempt feature added in the Lc0 v0.30 release, we shared our plans to add a lichess bot for piece odds games. While allowing arbitrary piece odds poses several challenges, which aren’t resolved yet, we are proud to announce that we made a big first step towards that goal with [LeelaKnightOdds](https://lichess.org/@/LeelaKnightOdds), now accepting your challenge on lichess.
+<!--more-->
+
+#### How can I play against LeelaKnightOdds?
+
+To challenge [the bot](https://lichess.org/@/LeelaKnightOdds) to a knight odds game, follow these steps:
+1. Click on the two crossed swords symbol in the top right to issue a challenge.
+2. Select the game variant “From Position” in the dropdown menu.
+3. Paste the starting FEN into the field;\
+`rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/R1BQKBNR w KQkq` to play without the knight on b1, and\
+`rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKB1R w KQkq` to play without the knight on g1.
+4. Select your preferred time control; only challenges with a main time between 1’ and 15’ and an increment between 1” and 10” will be accepted.
+5. To start the challenge, select black; games will be unrated independent of your selection.
+
+You don’t need a lichess account to challenge LeelaKnightOdds; note however that Leela will currently only play one game at the same time, and logged in lichess users might be given a higher priority.
+
+Unfortunately, the automatic “rematch” feature doesn’t work, as it will send an offer with switched colors. Just send a new challenge instead.
+
+The bot can also be challenged through the lichess [featured bots](https://lichess.org/player/bots) page, reachable from their top menu under "Community" &rarr; "Players" and then selecting "Online bots" on the side menu.
+
+#### How strong do I have to be to challenge LeelaKnightOdds?
+
+While the bot won’t reject challenges from players based on their rating, the primary target group are players rated above 2400 in blitz or rapid, and is estimated to perform around 2600 in 5’+3” games. Don’t hesitate though to give it a few tries even if you are lower rated, at worst you’ll get some first-hand experience of being fawned by Leela :smiley:. Generally, longer time controls should feel a bit easier.
```

Comment:
Wouldn't just typing 😊 work?

Reply:
You mean the unicode character, `&#x1f600;` or `:)`?

Reply:
The first two options work.